src/redux/rockets/rocketsSlice.js

```diff
@@ -0,0 +1,41 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+const url = 'https://api.spacexdata.com/v4/rockets';
+
+const initialState = {
+  rockets: [],
+  status: false,
+};
+
+export const fetchRockets = createAsyncThunk('rockets/fetchRockets', async () => {
+  const response = await axios.get(url);
+  return response.data;
```

Comment:
- [ ] Storing the entire data fetched from the API in Redux may result in unnecessary overhead and slow down the application. It would be great to only store the information necessary for this application. Kindly map the incoming `response.data` to store the required fields suggested in the project requirements. 😎️

**Project requirements**
![Screenshot from 2023-03-09 13-03-45](https://user-images.githubusercontent.com/17243325/223990957-ee6639f4-0dcf-4062-aa4b-c289fbb2c79a.png)